src/widget/woverview.cpp

```diff
@@ -901,17 +904,19 @@ void WOverview::drawTimeRuler(QPainter* pPainter) {
     QPen shadowPen(Qt::black, 2.5 * m_scaleFactor);
 
     if (m_bTimeRulerActive) {
-        QLineF line;
-        if (m_orientation == Qt::Horizontal) {
-            line.setLine(m_timeRulerPos.x(), 0.0, m_timeRulerPos.x(), height());
-        } else {
-            line.setLine(0.0, m_timeRulerPos.x(), width(), m_timeRulerPos.x());
-        }
-        pPainter->setPen(shadowPen);
-        pPainter->drawLine(line);
+        if (!m_bDrag) {
```

Comment:
I think this would be better handled in the mouse event code rather than the drawing code. `m_bTimeRulerActive` should not be set to true while dragging.

Reply:
It should. That is IMHO a feature. I only messed something up when committing so it was broken. 
Now it is working again.